### PR TITLE
feat: redpacket thegraph failover

### PR DESCRIPTION
**Pull request by hanszhoou**
closes #

### CODE REVIEW

File: packages/maskbook/src/plugins/RedPacket/Worker/database.ts [hunk 1]
```diff
@@ -21,23 +24,6 @@ export async function addRedPacket(record: RedPacketRecord) {
     return RedPacketDatabase.add(RedPacketRecordIntoDB(record))
 }
 
-export async function updateV1ToV2(history: RedPacketHistory) {
```
Comment (Jack-Works) [inline packages/maskbook/src/plugins/RedPacket/Worker/database.ts:24]: why?

**Comment by zhouhanseng** [comment]
@DimensionDev/qa

**Comment by zhouhanseng** [comment]
@Jack-Works Remove `updateV1ToV2()`, since all v1 red packets had expired(unable to claim) long time ago, and the password only used for user to claim.

File: packages/maskbook/src/plugins/RedPacket/Worker/database.ts (hunk 1 above)
Comment (Jack-Works) [reply packages/maskbook/src/plugins/RedPacket/Worker/database.ts:24]: I guess you should delete old records if you don't want them anymore.

**Pull request closed without merging**